charts/src/main/java/hu/ma/charts/pie/Utils.kt

```diff
@@ -29,6 +31,20 @@ internal fun PieChartData.createLegendEntries(
     )
   }
 
+internal fun LineChartData.createLegendEntries(
```

Comment:
Let's move this whole file into `utils` package, since it doesn't really fit here anymore

Reply:
Will fix